file-formats/sxtg/examples/sxtg_tsm_so.sxtg.json

```
@@ -0,0 +1,17 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "SXTG: Sales Order Header (Home)",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "extensionInclude": "SXTG_S_TSM_SO_INCL_EEW",
+    "applicationDataStructure": "SXTG_S_TSM_SO_APPL_DATA",
+    "uiExtensionEntity": "D_SXTG_TSM_SO_UI_HEADER"
+  },
+  "transactionCodes": [
+    {
+      "transactionCode": "SXTG_TSM_SO"
+    }
+  ]
```

Minor: In other object types we did not repeated the array name in the structure and changed it the field for the object name just to `name`. However, I would suggest to double check this in the UX review. 

This suggestion shall show how it could look like. I would wait whether our UX colleagues suggest to change it:

```suggestion
  "transactionCodes": [
    {
      "name": "SXTG_TSM_SO"
    }
  ]
```